tests/test_api.py

```
@@ -318,3 +318,67 @@ def test_openapi_schema(self, client):
+        # Add a few anomalies
+        for i in range(5):
+            telemetry = {
+                "voltage": 6.0 + i * 0.1,  # Slightly different values
+                "temperature": 50.0,
+                "gyro": 0.3
+            }
+            client.post("/api/v1/telemetry", json=telemetry)
+
+        # Get the first anomaly's voltage for verification
+        if len(anomaly_history) > 0:
+            first_anomaly_voltage = list(anomaly_history)[0].anomaly_score
```

Ruff (0.14.10)

Local variable `first_anomaly_voltage` is assigned to but never used

Remove assignment to unused variable `first_anomaly_voltage`

(F841)